config/secrets.yml

```diff
@@ -10,13 +10,22 @@
 # Make sure the secrets in this file are kept private
 # if you're sharing your code publicly.
 
```

Comment:
@mbarnett @murny Could use a review of this PR so that I can use the `shared` stuff below in other work.